src/main/java/com/java_essential/kvirikadze/homework/lesson3/issue2/Ship.java

```diff
@@ -0,0 +1,24 @@
+package com.java_essential.kvirikadze.homework.lesson3.issue2;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import static java.lang.String.format;
+
+public class Ship extends Vehicle {
+    @Getter @Setter private String homePort;
```

Comment:
ставте все аннотации в столбик над полем или классом  
```suggestion
     @Getter
     @Setter
     private String homePort;
```